Geometry_Engine/Create/ShapeProfile.cs

```diff
@@ -109,17 +109,17 @@ public static BoxProfile BoxProfile(double height, double width, double thicknes
                 return null;
             }
 
-            if (Math.Sqrt(2) * thickness <= Math.Sqrt(2) * outerRadius - outerRadius - Math.Sqrt(2) * innerRadius + innerRadius)
-            {
-                InvalidRatioError("thickness", "outerRadius and innerRadius");
-                return null;
-            }
-
-            if (height <= 0 || width <= 0 || thickness <= 0 || outerRadius < 0 || innerRadius < 0)
-            {
-                Engine.Reflection.Compute.RecordError("Input length less or equal to 0");
-                return null;
-            }
+            //if (Math.Sqrt(2) * thickness <= Math.Sqrt(2) * outerRadius - outerRadius - Math.Sqrt(2) * innerRadius + innerRadius)
+            //{
+            //    InvalidRatioError("thickness", "outerRadius and innerRadius");
+            //    return null;
+            //}
+
```

Comment:
I think it's best to omit this change from this PR, leave this scope to the MEP Engine only for the supporting methods, and move this suggestion to #1909 for discussion with @IsakNaslundBh and co next week as this helps structures a lot.